controllers/api.js

```diff
@@ -247,4 +247,12 @@ router
                 });
             }
         });
+    })
+
+    .use(function(err, req, res, next){
+        if (err) {
+            console.log(err);
+            res.status(500);
+            res[req.rt]({ error: err });
```

Comment:
Duh my bad. Totally set [here](https://github.com/meddulla/oocharts-server/blob/fix/showapierrors/controllers/api.js#L39). Merging! Thanks for the PR!